src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -2155,8 +2155,12 @@ <h2 class="text--center">Balance sheet</h2>
 
                       <tr class="figures">
                           <td id="creditors-within-label">Creditors: amounts falling due within one year:</td>
-
-                          <td class="figure noteIndex" id="uk-busCreditorsCreditorsWithinOneYear_CY_ENDNoteIndexes"></td>
+                          
+                        {{ if .small_full_accounts.balance_sheet_notes.creditors_within_one_year }}
```

Comment:
Sorry for being picky but can we fix the indentation, please? 
It is quite a big file so we need to try to keep it as tidy as possible